build/Helix/NuGet.config

```diff
@@ -19,7 +19,7 @@
     <clear />
     <add key="NuGet official package source" value="https://nuget.org/api/v3/" />
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="WindowsES-External" value="https://microsoft.pkgs.visualstudio.com/_packaging/WindowsES-External/nuget/v3/index.json" />	
```

Comment:
why this change?

Reply:
I renamed the package have Microsoft. in the front and I added it to ProjectReunion-Dependencies.
Now we can unlist it from there and pull it from WindowsES-External instead.